pkg/pmem-csi-driver/parameters.go

```diff
@@ -0,0 +1,272 @@
+/*
+Copyright 2019 Intel Corporation
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package pmemcsidriver
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/docker/go-units"
+)
+
+type persistencyModel string
+type parameterOrigin int
+
+// Beware of API and backwards-compatibility breaking when changing these string constants!
+const (
+	parameterCacheSize        = "cacheSize"
+	parameterEraseAfter       = "eraseafter"
+	parameterName             = "name"
+	parameterPersistencyModel = "persistencyModel"
```

Comment:
The `"persistencyModel"` parameter was a result of the initial thought of supporting different volume models using storage class, which I believe is not valid anymore. The `"cacheSize" `itself suffices for differentiating a cache volume. So we better to deprecate this parameter.

Reply:
So you are proposing to deprecate `parameterPersistencyModel` and instead say that any volume with `cacheSize` > 1 is a cache volume?

That could work.

Reply:
Let's discuss and implement this separately, i.e. not make that change in this PR.

Reply:
No problem to take it as separate PR, As this code is touching parameters so I thought its the right time to deprecate persistency model param.